### ISSUE TITLE
NVDA not announcing aria-label when async call happens on Chrome

### ISSUE THREAD

**Issue by svenka22** [nvaccess/nvda#8993]
### Steps to reproduce:
- Go to https://login.microsoftonline.com on Chrome
- Have NVDA running
- Click the ellipsis (...) on the right end of the footer 
- NVDA will announce the the aria-label for the link
- Click 'Enable' on the small pop-up that opens
- NVDA does not announce the aira-label when the link changes after Ajax call

Pls find the video of the same
[Aria-label not announced.zip](https://github.com/nvaccess/nvda/files/2630804/Aria-label.not.announced.zip)
### Actual behavior:
- NVDA does not announce the aira-label when the link changes after Ajax call

This is working fine on Microsoft Edge. Chrome has this problem
### Expected behavior:
- NVDA should announce the aira-label when the link changes after Ajax call
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2017.1
#### Windows version:
Windows 10
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by ehollig** [comment]
@svenka22 have you tried this with NVDA 2018.3 or 2018.4 beta?

**Comment by Adriani90** [comment]
cc: @derekriemer, @feerrenrut

**Comment by Adriani90** [comment]
@Qchristensen I am not sure if I tested it correctly because the steps to reproduce are very visual. I think I cannot reproduce it in Chrome 79 with NVDA 2019.3 Beta 1. Could you please test and report back? Thanks.

**Comment by Qchristensen** [comment]
The original steps are written very visually, but playing with it, this one is actually easy to repro with the keyboard:

1. Load https://login.microsoftonline.com/ in Chrome (load in an incognito window if, like me, your Chrome automatically logs you in to a Microsoft account).
2. The focus starts in the sign in edit.  Press ESCAPE to exit focus mode.
3. Press CONTROL+END to move to the end of the document.  This is the elipses button mentioned originally and it has the succinct label "link    Terms of use  link    Privacy & cookies  button  collapsed    Click here for troubleshooting information"
4. Press SPACEBAR to activate that button.
5. Some more information is displayed on the page, the focus moves to an advanced diagnostics link with the text "Disable".
6. Press SPACEBAR.  The link text changes to "Enable", but nothing is reported.

Pressing SPACEBAR at this point toggles the link text between enable and disable with no utterance from NVDA.

If you read the current line (NVDA+up arrow / NVDA+l) NVDA reads the correct text (enable or disable as appropriate).

**Comment by Adriani90** [comment]
This works as expected with NVDA Version: alpha-20015,56a62097 and Google Chrome or Edge Chromium 81.
Closing as works for me.
